tests/python_tests/test_eltwise_unary_sfpu.py

```diff
@@ -34,7 +38,15 @@ def generate_golden(operation, operand1, data_format):
     all_format_combos,
     dest_acc=[DestAccumulation.No, DestAccumulation.Yes],
     approx_mode=[ApproximationMode.No, ApproximationMode.Yes],
-    mathop=[MathOperation.Sqrt, MathOperation.Log, MathOperation.Square],
+    mathop=[
```

Comment:
Same can be done for mathop map 😁
```suggestion
        @classmethod
        def operations(cls):
            return [op for op in cls if op._function is not None]
```
Here's an example
`mathop = MathOperation.operations()`

Reply:
In principle, I agree, but at the moment our MathOperation contains a mix of unary and binary ops. And these can't be mixed. Maybe we should split the ops first?

Reply:
yes I agree as well, sounds good!